istioctl/cmd/analyze.go

```diff
@@ -324,32 +326,84 @@ istioctl analyze -L
 			`You can include the wildcard character '*' to support a partial match (e.g. '--suppress "IST0102=DestinationRule *.default" ).`)
 	analysisCmd.PersistentFlags().DurationVar(&analysisTimeout, "timeout", 30*time.Second,
 		"the duration to wait before failing")
+	analysisCmd.PersistentFlags().BoolVarP(&recursive, "recursive", "R", false,
+		"If true, recurse into directories when a directory is provided as an argument")
```

Comment:
Nit: Consider using nearly the same text as Kubectl, avoiding "If True" and "is provided as an argument".  It may be different because we don't insist on `--filename`.

kubectl says `Process the directory used in -f, --filename recursively. Useful when you want to manage related manifests organized within the same directory.`

Reply:
Went with: 

`Process directory arguments recursively. Useful when you want to analyze related manifests organized within the same directory.`